bin/ota-apply-technical-upgrades.js

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env node
+import './env.js';
+
+import path from 'path';
+import { fileURLToPath, pathToFileURL } from 'url';
+
+import { program } from 'commander';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const { applyTechnicalUpgrades } = await import(pathToFileURL(path.resolve(__dirname, '../src/index.js'))); // load asynchronously to ensure env.js is loaded before
+
+program
+  .name('ota apply-technical-upgrades')
+  .description('Apply technical upgrades: regenerate versions from existing snapshots with updated declarations/engine, and fetch missing snapshots for newly added source documents')
```

Comment:
`with updated declarations/engine`should be clarified if it is declarations and/or engine